src/main/java/by/georgprog/epicmusicstore/model/AlbumEntity.java

```diff
@@ -0,0 +1,61 @@
+package by.georgprog.epicmusicstore.model;
+
+import by.georgprog.epicmusicstore.model.user.UserEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity(name = "Album")
+@Table(name = "albums")
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+public class AlbumEntity {
+
+    @Id
+    @SequenceGenerator(
+            name = "albums_sequence",
+            sequenceName = "albums_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "albums_sequence"
+    )
+    @Column(
+            name = "id",
+            updatable = false
+    )
+    private Long id;
+
+    @Column(
+            name = "name",
+            nullable = false,
+            columnDefinition = "TEXT",
```

Comment:
Why do you use `TEXT` as column difinition here? This field already described as `varchar` through liquibase, we don't need this parameter here and in all string entity fields